stdlib/range.gr

```diff
@@ -126,3 +111,106 @@ provide let map = (fn, range) => {
   }
   result
 }
+
+provide module Inclusive {
+  /**
+   * Checks if the given number is within the range.
+   *
+   * @param value: The number being checked
+   * @param range: The range to check within
+   * @returns Whether or not the value is within range
+   *
+   * @example Range.Inclusive.inRange(1, { rangeStart: 0, rangeEnd: 1 }) == true
+   * @example Range.Inclusive.inRange(10, { rangeStart: 0, rangeEnd: 2 }) == false
+   *
+   * @since v0.6.0
+   * @history v0.3.0: Root APIs originally handled Inclusive & Exclusive variants
+   */
+
```

Comment:
Can the docs touch the implementations or is this a formatter bug?

Reply:
Formatter bug. We're not blocking on it though. Hopefully it'll be fixed soon.